build.sbt

```diff
@@ -66,26 +60,34 @@ lazy val compat = MultiScalaCrossProject(JSPlatform, JVMPlatform, NativePlatform
       name := "scala-collection-compat",
       moduleName := "scala-collection-compat",
       scalacOptions ++= Seq("-feature", "-language:higherKinds", "-language:implicitConversions"),
-      unmanagedSourceDirectories in Compile += {
-        val sharedSourceDir = (baseDirectory in ThisBuild).value / "compat/src/main"
+      Compile / unmanagedSourceDirectories += {
+        val sharedSourceDir = (ThisBuild / baseDirectory).value / "compat/src/main"
         if (scalaVersion.value.startsWith("2.13.")) sharedSourceDir / "scala-2.13"
         else sharedSourceDir / "scala-2.11_2.12"
       },
-      Test / sourceDirectories += (ThisBuild / baseDirectory).value / "compat/src/test/scala-jvm"
     )
     .jvmSettings(
-      junit
+      Test / unmanagedSourceDirectories += (ThisBuild / baseDirectory).value / "compat/src/test/scala-jvm",
```

Comment:
it's not clear to me why `sourceDirectories` doesn't work/what it does, but if this fixes it, great

Reply:
same here :-)